docs/content_management/taxonomy/taxonomy.md

```diff
@@ -36,6 +36,17 @@ To do it, first, create a new Content Type with `content_category` identifier an
 
 Next, in `config/packages/ibexa_taxonomy.yaml` add the following configuration:
 
+``` yaml
+[[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml', 1,16 )=]]
```

Comment:
Any reason to clip the `ibexa_taxonomy` key?

Reply:
Oh, probably didn't render, as it's in the code sample file. Will fix it.